components/designSystem/SmolAddressInput.tsx

```diff
@@ -42,7 +45,7 @@ export function SmolAddressInput(): ReactElement {
 		[getAddressBookEntry, value]
```

Comment:
Does it make sense to use `value` here? I think we could either JSON.stringify it or go with useDeepCompareMemo, wdyt?

Reply:
Ok, please merge this, would love to test smth too